.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+on:
+  push:
+    branches: [ master, '0.[0-9]+', 'v0.[0-9]+' ]
```

Comment:
`this-is-not-master` will not match, because we are not using regex in `master`.
This makes `master` an exact match.

For the second one, yes you are probably right.
Additionally I think we should change that to match any version apart from `0.x`.

Done in 04bf29508c5c98be51217e3ecbce58a1fa6994fd